src/main/java/com/todoist/pojo/ViewOption.kt

```diff
@@ -0,0 +1,51 @@
+package com.todoist.pojo
+
+open class ViewOption(
```

Comment:
What do you think about having interface similar to `UserPlan`? It will allow to have data class when implementing.